src/api/src/http/error.rs

```diff
@@ -95,8 +93,8 @@ impl ResponseError for Error {
         // error cannot happen.
         // A test covering all the possible error types are given to ensure this.
         let body = serde_json::to_string(&info).expect("serialize error response failed");
-        
-        // Due to the definition of KBS attestation protocol, 
+
+        // Due to the definition of KBS attestation protocol, we set the http code.
```

Comment:
I think that belongs to the HTTP error handling improvement commit (9cccf7ab4b3dca86fa2172a26bb1da1c85bb299f)

Reply:
Oh yes! Have rebased. Thanks!